ts/src/index.ts

```diff
@@ -61,5 +63,6 @@ export class KanmonPlatformApi {
     this.offers = new OffersApi(this._configuration);
     this.payments = new PaymentsApi(this._configuration);
     this.prequalifications = new PrequalificationsApi(this._configuration);
+    this.integratedMCAReceivablesApi = new IntegratedMCAReceivablesApi(this._configuration);
```

Comment:
Can we remove the `Api` suffix off the end of this like the others? That way it follows the convention?

Reply:
Looking at Java, they all have an Api suffix. I think we should keep this consistent between SDKs. @danielwong2268 which way do you prefer? Be our tie breaker!

Reply:
yas!

Reply:
Good catch, changed as suggested.